mobile/src/board/repository.tsx

```diff
@@ -0,0 +1,11 @@
+import { Board } from "./entity";
+
+export class BoardRepository {
```

Comment:
A static class with stateless functions is unusual in React, where we favor functional programming. Js already has the notion of modules, so you don't need static functions. I'd just export the `getById` function.